sdk/azidentity/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ### Features Added
 
 ### Breaking Changes
+> These changes affect only code written against a beta version such as v1.3.0-beta.5
+* Renamed `DisableInstanceDiscovery` to `DisableAuthorityValidationAndInstanceDiscovery`
```

Comment:
The length of the name hurts my head, but I don't think we can shorten it in any meaningful way.

Reply:
Well, what if we named it `DisableAuthorityValidation` or do we lose some important meaning?

Reply:
I suppose it could confuse someone wanting to disable instance discovery who doesn't know authority validation implies instance discovery. One goal of this new name is to make that relationship more explicit. It is a daunting name though 😞